3-extra/1-count-words.js

```diff
@@ -26,11 +26,29 @@
 function countWords(string) {
   const wordCount = {};
 
-  // write code here
+  if (!string || string.trim().length === 0) {
+    return wordCount;
+  }
+  let countWords = string.split(" ");
+
+  for (let word of countWords) {
+    wordCount[word] = count(word, countWords);
+  }
 
   return wordCount;
 }
 
+function count(word, countWords) {
+  let countWord = 0;
+
+  for (let wordCompare of countWords) {
+    if (wordCompare === word) {
+      countWord++;
+    }
+  }
+  return countWord;
+}
```

Comment:
Good, this works well. There are different ways of doing this so maybe compare with other trainees and with the solution, but your method is clear and concise, and I like how you spilt it up with a submethod.